test/js/test.js

```diff
@@ -136,22 +141,23 @@ function makeKeyPair() {
     state.certPrivateKey = cryptoUtil.importPemPrivateKey(fs.readFileSync(state.keyFile));
 
     console.log();
-    makeAccountKeyPair()
+    makeAccountKeyPair("account-key.pem", register)
   });
 }
 
-function makeAccountKeyPair() {
+function makeAccountKeyPair(keyName, callback) {
   console.log("Generating account key pair...");
-  child_process.exec("openssl genrsa -out account-key.pem 2048", function (error, stdout, stderr) {
-    if (error) {
-      console.log(error);
-      process.exit(1);
-    }
-    state.accountKeyPair = cryptoUtil.importPemPrivateKey(fs.readFileSync("account-key.pem"));
-    state.acme = new Acme(state.accountKeyPair);
+    child_process.exec("openssl genrsa -out " + keyName + " 2048", function (error, stdout, stderr) {
```

Comment:
A quick test shows we would have to add another step to find the actual path of `openssl` since `execFile` doesn't resolve names to their actual path using `PATH`.